### PR TITLE
Added custom path for rustup

### LINKED ISSUE #160: Custom rustup location
Hi.

In Windows 10 there is a subsystem called [WSL][1]. 
I want to use `rust` installed on Linux, while still working on Windows.

To do so I need to be able to pass custom `rustup` path to `rls-vscode` .

[1]: https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux

### PR DESCRIPTION
**Pull request by yisonPylkita**
Closes #160

### CODE REVIEW

File: src/configuration.ts [hunk 1]
```diff
@@ -44,14 +45,15 @@ export class RLSConfiguration {
      * directory.
      */
     public readonly rlsRoot: string | null;
-
+
```
Comment (nrc) [inline src/configuration.ts]: nit: please don't add whitespace

File: src/configuration.ts [hunk 2]
```diff
@@ -44,14 +45,15 @@ export class RLSConfiguration {
...
-
+    
     public static loadFromWorkspace(): RLSConfiguration {
         const configuration = workspace.getConfiguration();
 
         return new RLSConfiguration(configuration);
     }
 
     private constructor(configuration: WorkspaceConfiguration) {
+        this.rustupPath = configuration.get('rustup.path', 'rustup');
```
Comment (nrc) [inline src/configuration.ts]: Could you use something like`rust-client.rustupPath` for the configuration name please? Could you also add the option to the `configuration` section of package.json.

File: src/configuration.ts (hunk 1 above)
Comment (yisonPylkita) [reply src/configuration.ts]: Ok, I will correct this and pay attention to this in future.

File: src/configuration.ts (hunk 2 above)
Comment (yisonPylkita) [reply src/configuration.ts]: Yeah, you are right.

I will submit patch when I get home.

**Comment by nrc** [comment]
Looks good, thanks! Could you rebase please?

**Comment by yisonPylkita** [comment]
Sorry for the delay.

I have updated this branch to an actual master. Is this a proper way of doing this ?

**Comment by yisonPylkita** [comment]
Sorry for the delay. 
I have rebased this branch onto actual master.

**Comment by Xanewok** [comment]
Thank you!

**Pull request merged**